wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/FwLogParsedMsg.java

```diff
@@ -17,6 +17,7 @@ public void close() {
     public String getSeverity() {return nGetSeverity(mHandle);}
     public int getLine() {return nGetLine(mHandle);}
     public long getTimestamp(){return nGetTimestamp(mHandle);}
+    public int getSequence() {return nGetSequenceId(mHandle);}
```

Comment:
Shouldn't it be `getSequenceId(..)`  for symmetry?

Reply:
done